styles.css

```diff
@@ -0,0 +1,491 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  background-color: #e5e5e5;
+}
+
+.hero {
+  background-image: url(assets/images/background-shapes.png);
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+  min-height: 587.15px;
+  border-radius: 0 0 0 15%;
+  padding: 0 24px;
+  background-color: #fff;
+  background-position: center;
+  margin-bottom: 114px;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 12px 24px;
+}
+
+.my-logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.nav-menu {
+  display: none;
+}
+
+.greeting {
+  padding-top: 128px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.job-description {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  color: #344563;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  margin-top: 0.75rem;
+}
+
+.connect-link {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  color: #7f8cff;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 1rem;
+  line-height: 0.875rem;
+  margin: 0.75rem 0 0.875rem 0;
+}
+
+li a {
+  display: block;
+  text-decoration: none;
+}
+
+.social-media-links a {
+  margin-right: 1.188rem;
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+}
```

Comment:
- I noticed that the stack from the project looks squashed in on the mobile screen and breaks the text, You can use a `flex wrap` property on the `ul` to make it look on on the mobile screen. Check out this [article](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap) to understand what to do, in other to fix it. 

##### Your design 
![image](https://github.com/znation2010/Mobile_Version_Portfolio/assets/58771507/469385e0-4bd9-4d80-bdf8-85ec598f6513)